tests/unit/amo/components/TestSearchForm.js

```diff
@@ -88,21 +113,60 @@ describe('<SearchForm />', () => {
     expect(router.push.called).toBeTruthy();
   });
 
+  it('passes addonType when set', () => {
+    root = renderIntoDocument(
+      <SearchFormWrapper addonType={ADDON_TYPE_EXTENSION} />
+    ).root;
+    form = root.form;
+    input = root.searchQuery.input;
+
+    expect(!router.push.called).toBeTruthy();
+    input.value = '& 26 %';
+    Simulate.click(root.submitButton);
+    expect(router.push.calledWith({
+      pathname: '/de/firefox/search/',
+      query: { q: '& 26 %', type: ADDON_TYPE_EXTENSION },
+    })).toBeTruthy();
+  });
+
   it('encodes the value of the search text', () => {
     expect(!router.push.called).toBeTruthy();
     input.value = '& 26 %';
     Simulate.click(root.submitButton);
     expect(router.push.calledWith({
       pathname: '/de/firefox/search/',
-      query: { q: '& 26 %' },
+      query: { q: '& 26 %', type: undefined },
     })).toBeTruthy();
   });
 });
 
 describe('SearchForm mapStateToProps', () => {
   it('passes the api through', () => {
-    const api = { clientApp: 'firefox', lang: 'de', token: 'someauthtoken' };
-    expect(mapStateToProps({ foo: 'bar', api })).toEqual({ api });
+    const { store } = dispatchSignInActions();
+    store.dispatch(setViewContext(VIEW_CONTEXT_HOMEPAGE));
```

Comment:
No it's not. Must have added it when I moved to the dispatch helper as other tests used it. Will remove.